protocol/chat.go

```diff
@@ -31,6 +31,8 @@ var chatColors = []string{
 
 type ChatType int
 
+type ChatContext string
```

Comment:
does this need to be public? seems to be only used in `protocol` package.
```suggestion
type chatContext string
```

Reply:
Yes, to my knowledge it should be possible normally to use `chatContext` instead of `ChatContext` to make sure the type isn't exported from the package, however, I get a golint error 😢  It must be a misconfiguration in the linting, I suspect the files in the package are compile independently or something. Is this blocking?

Reply:
Weird, but not blocking of course 🙂